.gitignore

```diff
@@ -6,6 +6,7 @@ tags
 
 ## Ignore project files created by Eclipse
 .settings/
+bin/
```

Comment:
As @tomdesair mentions, this will exclude changes to subdirectories named "bin" as well.

In order to just exclude a top level "bin" directory, I think this needs to be `/bin/`